### DISCUSSION TITLE
Some enhances for the Studio

### DISCUSSION THREAD

**Discussion by mdre** [ArcadeData/arcadedb#1147] [Ideas]
I'm using Studio a lot and there some things that could be nice to have:

- [ ] In the studio, in tab-database, it is really important to have a way to search for a type. In Orient we have a filter at the top to search. Here there's a really looooong list on the left and if I scroll I lost the fields view.
- [x] When I click over a type it show only the direct type field but not the inherited as Orient do.
- [x] If you click on "super type" to see it, you can't get back to the previous type.
- [x] it really important to have a way to store the querys. A Load/Save button could be help. The right top button open to the right and if the browser is maximize it go off-screen.
- [x] Since when I work it's not clear the query I want, it is usefull to excecute part of a query just highligting it and pressing run. 
- [ ] Preserve the output of a query like Orient do or in tab like others database.
- [ ] Highlight matched brackets.
- [ ] In the Graph view, when you want to navigate through an edge, it add all of it and not let you choose which to add as Orient do.

**Comment by lvca** [comment]
Just pushed this: https://github.com/ArcadeData/arcadedb/commit/32661317dec781541a0cdd7f2759de57e312049a: "Following the request on https://github.com/ArcadeData/arcadedb/discussions/1147, now also sub-types are displayed, so it's easier to go back and forth between types following the hierarchy. Also, now all the properties are displayed, included the one from super types."

**Comment by lvca** [comment]
Also moved the History button in a better position:

https://github.com/ArcadeData/arcadedb/commit/47be5dba00ab1b6f2912689078e7deb09bb83733

"Following the request on https://github.com/ArcadeData/arcadedb/discussions/1147, the history now is more visible and does not disappear on window resizing."

**Comment by lvca** [comment]
Execution of selected text has been implemented in 0b624b183.
